### PR TITLE
Fix font weights

### LINKED ISSUE #813: Bold tekst når verdi på select mangler på flere komponenter
**Describe the bug**
Når man har gjort et valg skal tekst vises som bold slik det nå er på Radiobuttons.
Dette fungerer ikke i checkbokser og compact-versjoner av datepicker, select og kompakte inputfelter. 
Trenger en gjennomgang av alle komponenter for å sikre konsistens.

Kan være noen avvik mellom skisser i Figma og kode og delvis inkonsistens mellom komponentene

**Expected behavior**
Bold tekst på select

**Screenshots**
764cd3fc-875d-49a2-94c6-5c5debb16079

f54954ea-838b-4b35-ab31-8e309ec806f0

6bdf02c2-e83d-498e-934a-723993b38112

5c21b7c5-8533-4272-bfde-8b7eb37d0590

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

Fix bugs where wrong font weight was used for some variants of the components

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

Closes #813 

**Edit**:
This update uncovered a bug that might lurk in most of our components: Since we are using a `<label>` tag as a wrapper for entire components, such as `TextField`, `Checkbox` and `RadioButtonOption`, we might break HTML standards by having interactive elements inside a `<label>` tag. This happens in `ActionTextField`, and used to be the case in `Datepicker`.

I propose that we move away from this pattern entirely to avoid this bug, and refactor all components using the pattern ASAP.

### CODE REVIEW

File: packages/checkbox-react/src/Checkbox.tsx [hunk 1]
```diff
@@ -29,9 +30,10 @@ export function Checkbox({
         "jkl-checkbox--inline": inline,
         "jkl-checkbox--error": invalid,
     });
+    const [uuid] = useState(`jkl-checkbox-${nanoid(8)}`);
```
Comment (joms) [inline packages/checkbox-react/src/Checkbox.tsx]: This is really not a uuid 🤔

Comment (lfbergee) [reply packages/checkbox-react/src/Checkbox.tsx]: Burde ikke bli flere renders, state blir initialisert med en (ikke uuid, men unik nok sannsynligvis) id, den blir aldri endret, så innholdet burde overleve eventuelle rerendenders.

Comment (piofinn) [reply packages/checkbox-react/src/Checkbox.tsx]: Hehe, tror "uuid" sitter litt i fingrene bare 😅 Kan gjerne kalle den noe annet (bare `id`?) for korrekthetens skyld. Tror heller ikke det skal bli ekstra rerenders med state?

Comment (piofinn) [reply packages/checkbox-react/src/Checkbox.tsx]: > men unik nok sannsynligvis

Skjermbilde 2020-03-10 kl  15 49 07
😁

**Pull request merged**